composer.json

```
@@ -1,5 +1,5 @@
 {
-    "name": "gerardojbaez/money",
+    "name": "VBMart/money",
```

This needs to be changed to the old package name `gerardojbaez/money`

```suggestion
    "name": "gerardojbaez/money",
```